### PR TITLE
Fix outputs of the dist commands

### PR DESCRIPTION
**Pull request by pubudu91**

### CODE REVIEW

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public String getName() {
 
     @Override
     public void printLongDesc(StringBuilder out) {
-        out.append("Updates the Ballerina cli commands to latest version. \n");
+        out.append("Updates the Ballerina CLI commands to the latest version.\n");
```
Comment (keizer619) [inline ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java]: @pubudu91 ; Lets refer this as "Ballerina command"

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java [hunk 2]
```diff
@@ -88,10 +88,10 @@ public void setParentCmdParser(CommandLine parentCmdParser) {
 
     private static void updateCommands(PrintStream printStream) {
         String version = ToolUtil.getCurrentToolsVersion();
-        printStream.println("Fetching latest version from remote server...");
+        printStream.println("Fetching the latest version from the remote server...");
         String latestVersion = ToolUtil.getLatestToolVersion();
         if (latestVersion.equals(version)) {
-            printStream.println("You are already in latest command version: " + latestVersion);
+            printStream.println("The command is already in the latest version: " + latestVersion);
```
Comment (pubudu91) [inline ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java]: Don't we have a proper term to refer to the tool?

Comment (pubudu91) [reply ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java]: @keizer619 @Kishanthan

Comment (keizer619) [reply ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java]: @pubudu91 I think It should be referred as "command". With installers what user get is the ballerina command which has all the sub commands handled by a ballerina distribution. Ballerina distribution itself could be managed with "dist" subcommand.

File: ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java [hunk 3]
```diff
@@ -558,14 +560,12 @@ public static void handleInstallDirPermission() {
         try {
             String installationPath = OSUtils.getInstallationPath();
             if (!new File(installationPath).canWrite()) {
-                throw ErrorUtil.createCommandException("current user does not have permissions to run the command\n\n" +
-                                                               "Current user does not have write permissions to " +
-                                                               "Ballerina installation path " + installationPath +
-                                                               ". Grant permission to the file path or run the " +
-                                                               "command with user has write permission.");
+                throw ErrorUtil.createCommandException(
+                        "permission denied: current user does not have write access to the Ballerina installation " +
```
Comment (warunalakshitha) [inline ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java]: We had something similar to these. But there was requirement to tell what we need to do after saying permission have permission. Any +1 revert to short message.

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java [hunk 4]
```diff
@@ -62,19 +62,19 @@ public void execute() {
...
         }
         String distributionVersion = distribution.replace(distributionType + "-", "");
         if (distributionVersion.equals(ToolUtil.getCurrentBallerinaVersion())) {
-            printStream.println(distribution + " already in use");
+            printStream.println("'" + distribution + "' is the current distribution in use");
             return;
         }
         if (ToolUtil.downloadDistribution(printStream, distribution, distributionType, distributionVersion)) {
-            printStream.println(distribution + " is already fetched");
+            printStream.println("Distribution '" + distribution + "' is locally available");
```
Comment (MaryamZi) [inline ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java]:
```suggestion
            printStream.println("Distribution '" + distribution + "' is available locally");
```

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java [hunk 5]
```diff
@@ -62,19 +62,19 @@ public void execute() {
...
         } else {
-            printStream.println(distribution + " is fetched");
+            printStream.println("Distribution '" + distribution + "' fetched");
```
Comment (MaryamZi) [inline ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java]:
```suggestion
            printStream.println("Fetched distribution '" + distribution + "'");
```

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/ListCommand.java [hunk 6]
```diff
@@ -59,14 +59,14 @@ public void execute() {
...
-            throw ErrorUtil.createUsageExceptionWithHelp("too many arguments given");
-        }
-
         String userCommand = listCommands.get(0);
         if (parentCmdParser.getSubcommands().get(userCommand) == null) {
             throw ErrorUtil.createUsageExceptionWithHelp("unknown command " + userCommand);
         }
+
+        if (listCommands.size() > 1) {
+            throw ErrorUtil.createUsageExceptionWithHelp("too many arguments given");
```
Comment (MaryamZi) [inline ballerina-command/src/main/java/org/ballerinalang/command/cmd/ListCommand.java]:
```suggestion
            throw ErrorUtil.createUsageExceptionWithHelp("too many arguments");
```

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateCommand.java [hunk 7]
```diff
@@ -86,13 +87,13 @@ public void setParentCmdParser(CommandLine parentCmdParser) {
...
         String version = ToolUtil.getCurrentBallerinaVersion();
-        printStream.println("Fetching latest distribution version from remote server...");
+        printStream.println("Fetching the latest distribution version from the remote server...");
         String latestVersion = ToolUtil.getLatest(version, "patch");
         if (!latestVersion.equals(version)) {
             String distribution = ToolUtil.BALLERINA_TYPE + "-" + latestVersion;
             ToolUtil.downloadDistribution(printStream, distribution, ToolUtil.BALLERINA_TYPE, latestVersion);
             ToolUtil.useBallerinaVersion(printStream, distribution);
-            printStream.println("Updated to latest distribution version: " + latestVersion);
+            printStream.println("Updated to the latest Ballerina version: " + latestVersion);
```
Comment (MaryamZi) [inline ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateCommand.java]: Shouldn't this be 

```suggestion
            printStream.println("Updated to the latest Ballerina version: '" + latestVersion + "'");
```

for consistency?

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/ListCommand.java (hunk 6 above)
Comment (MaryamZi) [reply ballerina-command/src/main/java/org/ballerinalang/command/cmd/ListCommand.java]: Applicable elsewhere too.

File: ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java [hunk 8]
```diff
@@ -318,8 +320,8 @@ public static void checkForUpdate(PrintStream printStream, String[] args) {
                     String latestVersion = ToolUtil.getLatest(version, "patch");
                     if (!latestVersion.equals(version)) {
                         printStream.println();
-                        printStream.println("A new Ballerina version is available : " + latestVersion);
-                        printStream.println("You can download the installer of it from " +
+                        printStream.println("A new Ballerina version is available: " + latestVersion);
```
Comment (MaryamZi) [inline ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java]:
```suggestion
                        printStream.println("A new version of Ballerina is available: '" + latestVersion + "'");
```

File: ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java (hunk 3 above)
Comment (keizer619) [reply ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java]: @Kishanthan ; Any concerns?

Comment (pubudu91) [reply ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java]: A developer would know what to do if we say that the user doesn't have write access to the Ballerina installation directory right?

**Pull request merged**